src/jumpstart/JumpstartEnterAmount.tsx

```diff
@@ -167,10 +175,38 @@ function JumpstartEnterAmount() {
     jumpstartSendStatus === 'success' ||
     jumpstartSendStatus === 'loading'
 
+  const onIntroDismiss = () => {
+    dispatch(jumpstartIntroSeen())
+    replace(Screens.JumpstartEnterAmount)
+  }
+
+  // Track it in analytics whenever user sees intro screen for the first time
+  useEffect(() => {
+    if (!introSeen) {
+      AppAnalytics.track(JumpstartEvents.jumpstart_intro_seen)
+    }
+  }, [introSeen, dispatch])
+
   if (tokens.length === 0) {
     return <JumpstartAddAssets />
   }
 
+  if (shouldShowIntro) {
+    return (
+      <JumpstartIntro
+        button={
+          <Button
+            testID="JumpstartEnterAmount/haveFundsButton"
+            onPress={onIntroDismiss}
+            text={t('jumpstartIntro.haveFundsButton')}
+            type={BtnTypes.PRIMARY}
+            size={BtnSizes.FULL}
+          />
```

Comment:
the props of this component made me think that the consumer needs to know too much (it needs to create the button, and assign whether the assets hint needs to be displayed) - what do you think about combining the jumpstart intro and add assets screens? the benefit would be
- the JumpstartEnterAmount could render this component if the user hasn't seen it before or if they have no assets, with no props and no other coordination
- this component can decide whether to render the no funds hint and what to do with the button, since it can get this information from redux

maybe i'm missing some details here, i think my point of confusion comes from the flow of these components, where it seems like JumpstartEnterAmount needs to decide which component to render but the 2 of 3 options are actually related (one uses the other) under the hood.